packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -71,6 +71,23 @@ import pfLogoSmall from './pf-logo-small.svg';
 ```ts file='./CardSelectable.tsx'
 ```
 
+### Selectable accessibility highlight
+
+This example demonstrates how the `hasSelectableInput` and `onSelectableInputChange` props enable improved accessibility for selectable cards.
```

Comment:
@wolfeallison can you please review the copy here from lines 76-86.

Reply:
I think we could say "props improves accessibility for selectable cards." in line 76 and take out the "enable improved"

For line 78, I had a little trouble understanding what it was saying the first time and would switch the last sentence to " This input communicates to assistive technology users that a card is selectable, and if so, lists the current selection state." Not sure if "lists" is the right verb here or how the selection state is presented

For line 80, add a comma after "If you don't use the card title component in your selectable card,"

Reply:
@wolfeallison for line 78, thoughts on: "This input communicates to assistive technology users that a card is selectable, and if so, it communicates the current selection state as well."?

Reply:
@wise-king-sullyman I like that!